packages/suite/src/components/guide/GuideButton.tsx

```diff
@@ -35,12 +35,9 @@ const Wrapper = styled.button<{ guideOpen?: boolean; isModalOpen?: boolean }>`
 `;
 
 const GuideButton = () => {
-    const { isMobileLayout } = useLayoutSize();
     const { openGuide, guideOpen, isModalOpen } = useGuide();
     const analytics = useAnalytics();
 
-    if (isMobileLayout) return null;
```

Comment:
I just tested it. I would leave it on the welcome page, onboarding, maybe settings in onboarding. But it is not practical inside the app. @matejkriz

Reply:
To be honest, I don't know. I believe it was an open question. But since it's so easy to hide/show it, let's ask in #suite_ui for the feedback? 

I would vote for keeping it everywhere (even on dashboard and accounts pages).

Reply:
Will discuss with Ondra once again. In the menu it's not that visible - the question is how visible do we want it to be. 

It won't be in the modal headers, as it's planned to eventually use that place for guide anchors: #5062

Reply:
The button will be removed on mobile completely, the modal anchors are postponed and for now modals will have a guide button which just opens the guide.

Reply:
done